client/src/components/SignUpForm.js

```diff
@@ -97,6 +97,7 @@ const SignUpForm = () => {
       Auth.login(data.createUser.token);
       setSuccessMessage('Welcome!! Your are now signed up!')
       sendForm("service_k0uycid", "template_0jr5hbi", form.current, "NZ0ltP_Q1eOniKe9w")
+      window.location.assign('/')
```

Comment:
Not `.replace`?

Reply:
Assign was how it was originally in the Auth file. I removed it from their to add the history(-1) to login. I moved the assign from the Auth file to just the signup. Before it applied to login and signup